app/src/main/java/org/simple/clinic/navigation/v2/Router.kt

```diff
@@ -453,6 +440,24 @@ class Router(
       throw RuntimeException("Can only execute navigation state changes on the UI thread! Current thread is [${Thread.currentThread().name}].")
     }
   }
+
+  private fun getTransactionAnimations(
+      direction: Direction,
+      transactionOptions: TransactionOptions?
+  ) = when (direction) {
+    Direction.Forward -> Pair(
+        transactionOptions?.enterAnim ?: R.anim.router_slide_in_right,
+        transactionOptions?.exitAnim ?: R.anim.router_slide_out_left
+    )
+    Direction.Backward -> Pair(
+        transactionOptions?.enterAnim ?: R.anim.router_slide_in_left,
+        transactionOptions?.exitAnim ?: R.anim.router_slide_out_right
+    )
+    Direction.Replace -> Pair(
+        transactionOptions?.enterAnim ?: R.anim.router_fade_in,
+        transactionOptions?.exitAnim ?: R.anim.router_fade_out
+    )
```

Comment:
Another option we have is to add the default animation IDs as properties of the `Direction` enum and then we can do something like:

```kotlin
val defaultEnterAnim = direction.defaultEnterAnim
val defaultExitAnim = direction.defaultExitAnim

return when(direction) {
}
```

What do you think?

Reply:
We can do that, but we would be exposing `Direction` enum to public APIs. That's why went with a separate class since this can be used for other use cases in future to customise the transaction/navigation behaviour.

Reply:
We can probably move the `Direction` enum inside the `TransactionOptions` as well

Reply:
Ah cool. It was just a suggestion, whatever's here now works as well.

Reply:
Yeah 😅